hybrid/samplers.py

```diff
@@ -157,10 +155,11 @@ class ReverseAnnealingAutoEmbeddingSampler(Runnable, traits.SubproblemSampler):
         num_reads (int, optional, default=100):
             Number of states (output solutions) to read from the sampler.
 
-        qpu_sampler (:class:`dimod.Sampler`, optional, default=EmbeddingComposite(DWaveSampler())):
-            Quantum sampler such as a D-Wave system. If sampler is structured,
-            it will be converted to unstructured via
-            :class:`~dwave.system.composited.EmbeddingComposite`.
+        qpu_sampler (:class:`dimod.Sampler`, optional,
+                default=:class:`AutoEmbeddingComposite`(:class:`DWaveSampler`())):
+            Quantum sampler such as a D-Wave system. If the subproblem does not
+            fit the structure of the sampler, it will be auto-embedded on fly with
```

Comment:
Minor suggestion: ""Subproblems that do not fit the sampler's structure are minor-embedded on the fly..."
(also fixes small typo "on the fly")